compiler/src/main/java/io/neow3j/compiler/converters/MethodsConverter.java

```diff
@@ -194,6 +197,12 @@ private static AbstractInsnNode handleUncachedMethodCall(NeoMethod callingNeoMet
         return methodInsn;
     }
 
+    private static boolean isThrowableGetMessage(ClassNode owner, MethodInsnNode methodInsn) {
+        return getFullyQualifiedNameForInternalName(owner.name).equals(
+                Throwable.class.getCanonicalName()) &&
+                methodInsn.name.equals(THROWABLE_GET_MESSAGE);
```

Comment:
Wait, so the owner is `Throwable` and not `Exception` or `AssertionError`? That's convenient.

Reply:
Yes, both `AssertionError` and `Exception` use the method `getMessage` from their shared super class `Throwable`, which is super convenient. 🙂